### PR TITLE
PP3 CH13&14

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: piety-promoted-v3/modernized/13-chapter-13.adoc [hunk 1]
```diff
@@ -302,12 +301,11 @@ This is not often the case; but I desire to be content and thankful.`'
 
 She departed the 31st of the Fifth month, 1804.
 
-A short sketch of the engagement of mind of Elizabeth Millis,
-referred to in the foregoing account.
+=== A short sketch of the engagement of mind of Elizabeth Millis, referred to in the foregoing account.
```
Comment (maijapapaya) [inline piety-promoted-v3/modernized/13-chapter-13.adoc]: Not sure if this should be a sub heading or not.

File: piety-promoted-v3/modernized/13-chapter-13.adoc [hunk 2]
```diff
@@ -224,13 +223,13 @@ and took but little nourishment.
...
 and her bodily conflict was great.
-Some of her friends u:ho had retired, came again to her chamber.
+Some of her friends who had retired, came again to her chamber.
 She appeared glad to see them, and with earnestness said, "`Rejoice evermore,
 and in everything give thanks.`"
 Thankfulness, no doubt, reigned in her soul,
 in the expectation of a speedy relief from the oppressive state of her enfeebled frame.
 But this she yet was permitted to endure for a time.`'
-This, '`she said, '`is a hard conflict.
+This, '`she said,`' is a hard conflict.
```
Comment (jaredh159) [inline piety-promoted-v3/modernized/13-chapter-13.adoc]: I think we've got the wrong thing wrapped in single quotes here, I think everything but `she said,` should be quoted.

```ASCIIDOC
'`This,`' she said, '`is a hard conflict.
```

File: piety-promoted-v3/modernized/13-chapter-13.adoc (hunk 1 above)
Comment (jaredh159) [reply piety-promoted-v3/modernized/13-chapter-13.adoc]: sort of a tough call, but I think you got it right. 👍

File: piety-promoted-v3/modernized/13-chapter-13.adoc (hunk 2 above)
Comment (maijapapaya) [reply piety-promoted-v3/modernized/13-chapter-13.adoc]: haha, it's funny how I mess these up sometimes. I think these are the times there's too much noise in the house. 😄

**Pull request merged**